documentation/docs/libraries/wasm/cheat_sheet.md

```diff
@@ -151,7 +153,7 @@ Document.fromKeyPair(key:keyPair, network:string|undefined);
 ```
 </details>
 
-### [Publish](../../decentralized_identifiers/create.mdx)
+### [Publish](https://wiki.iota.org/identity.rs/decentralized_identifiers/create)
```

Comment:
Why change to absolute urls instead of relative?

Reply:
My mind automatically triggers "md or mdx" files as bad and changes them. I can switch them back 🤷

Reply:
IMO relative paths are better because we'll have less work if wiki url structure changes.